src/test/resources/org/kohsuke/github/GHContentIntegrationTest/wiremock/testGetFileContentWithNonAsciiPath/__files/repos_github-api-test-org_ghcontentintegrationtest_contents_ghcontent-ro_a-dir-with-nonascii-entry_bornsteintxt-3.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "B??????rnstein.txt",
+  "path": "ghcontent-ro/a-dir-with-nonascii-entry/B??????rnstein.txt",
```

Comment:
This is interesting.  What happens if you take this path string and call `repo.getFileContent()` on it?

Reply:
You mean against the real github? i haven't tried that but it surely won't work.

Reply:
It should work.  For other paths you can do exactly this.  It's a roundtrip thing.